Gopkg.toml

```diff
@@ -5,6 +5,7 @@ required = [
     "github.com/go-openapi/spec",
     "github.com/onsi/ginkgo", # for integration testing
     "github.com/spf13/pflag",
+    "github.com/pkg/errors",
```

Comment:
dep will drop it w/o this line even it's a dependency of sigs.k8s.io/controller-tools